### DISCUSSION TITLE
Carrier Disabled VoWIFI when Roaming, can we force enable it?

### DISCUSSION THREAD

**Discussion by xantrk** [kyujin-cho/pixel-volte-patch#52] [Q&A]
I'm using DUAL SIM. With this app I managed to enable VoWIFI on one of my SIMs smoothly.

As title says, my other carrier EE UK disables WIFI Calls when I'm abroad. Do you think it is possible to force it to be enabled when I'm roaming using this app?

Also, one of my operators allow me to have 5G toggle, backup calling (use data sim to make "wifi" calls from roaming sim) feature toggles but EE doens't. I was wondering if those are possible to forced to be enabled as well?

**Comment by peixotorms** [comment]
vpn maybe?
